classes/class-object-sync-sf-logging.php

```diff
@@ -95,6 +95,7 @@ public function __construct() {
 
 		$this->enabled         = filter_var( get_option( $this->option_prefix . 'enable_logging', false ), FILTER_VALIDATE_BOOLEAN );
 		$this->statuses_to_log = maybe_unserialize( get_option( $this->option_prefix . 'statuses_to_log', array() ) );
+		$this->statuses_to_log = empty( $this->statuses_to_log ) ? [] : $this->statuses_to_log;
```

Comment:
There are multiple flavors of this issue, but they all come down to this: when the option doesn't exist or is empty then returned value from `get_option` is an empty string. Then `maybe_unserialize` thinks this is an ok value and doesn't return the `array` set as the default.

Then array functions throw fatals. Ex: `in_array`